### ISSUE TITLE
Show error when repositories rescan fails

### ISSUE THREAD

**Issue by RaptorCZ** [ForkIssues/TrackerWin#1783]
I just cloned a new repository to disk. Tried to `rescan repositories` to get it to repos view. Nothing happens. Tried multiple times, tried more repos. The same problem. Then I tried to `Open repository` from menu. Then I got error

```
fatal: detected dubious ownership in repository at 'C:/Develope/Repos/XXX'
'C:/Develope/Repos/XXX' is owned by:
	'S-1-1-11-111'
but the current user is:
	'S-1-1-11-1111111111-1111111111-1111111111-1111'
To add an exception for this directory, call:

	git config --global --add safe.directory C:/Develope/Repos/XXX
```

Well, I still have no idea why this feature in git exists, but ok. 

Could you please throw any error from rescan, if error happens? Just to show user what happens. It took me some time to figure out, why my new repo is not found.

**Comment by DanPristupov** [comment]
Thank you for posting this.

That is a new behavior in git 2.39.1. We'll make Fork handle it in the next update.

**Comment by DanPristupov** [comment]
Fixed in Fork 1.82
